lib/5.26.2/pod/perltoc.pod

```diff
@@ -20144,146 +20144,6 @@ utf8, encode, compress, int32, null
 
 =back
 
-=head2 DB_File - Perl5 access to Berkeley DB version 1.x
```

Comment:
Missed installing Berkeley DB (and GDBM) during the rebuild, these need to be here...